scp.c

```diff
@@ -2077,9 +2083,17 @@ sink(int argc, char **argv, const char *src)
 		omode = mode;
 		mode |= S_IWUSR;
 #ifdef WINDOWS
-		if (add_mark_of_web(np) == -1) {
-			run_err("%s: failed to add mark of the web\n", np);
-			exit(1);
+		if (motw_zone_id == 5) {
+			if (verbose_mode)
```

Comment:
I don't think we need this verbose note anymore, since this is expected (no motw for push case).